src/main/java/org/sunbird/ratings/controller/RatingsController.java

```diff
@@ -0,0 +1,48 @@
+package org.sunbird.ratings.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import org.sunbird.common.model.SBApiResponse;
+import org.sunbird.ratings.model.RequestRating;
+import org.sunbird.ratings.service.RatingService;
+
+import javax.validation.Valid;
+
+@RestController
+public class RatingsController {
+
+    @Autowired
+    RatingService ratingService;
+
+    // ----------------- Public APIs --------------------
+
+    @PostMapping("/ratings/v1/upsert")
+    public ResponseEntity<?> upsert(@Valid @RequestBody RequestRating requestRatingBody) throws Exception {
```

Comment:
Should we just call this the update API?

Reply:
We are doing both addition and update in this so named it upsert to give more clarity about its working.